apps/omg_watcher_info/lib/omg_watcher_info/exit_consumer.ex

```diff
@@ -32,15 +32,16 @@ defmodule OMG.WatcherInfo.ExitConsumer do
   def init(args) do
     topic = Keyword.fetch!(args, :topic)
     bus_module = Keyword.get(args, :bus_module, @default_bus_module)
+    state = %{event_type: Keyword.fetch!(args, :event_type)}
 
     :ok = bus_module.subscribe(topic, link: true)
 
     _ = Logger.info("Started #{inspect(__MODULE__)}, listen to #{inspect(topic)}")
-    {:ok, %{}}
+    {:ok, state}
```

Comment:
Hmm. I think we misunderstood each other. By state war I meant keeping just event_type in state. 

{:ok, Keyword.fetch!(args, :event_type)}

Reply:
Ooch - it's clever, but it's quite uncommon in our code... I would leave it as it looks now if you not strongly against ❓

Reply:
Then make a struct. Which is the pattern thats most common.

Reply:
Or ignore me, no strong objections 😹